examples/simple_box/src/client.rs

```diff
@@ -98,10 +96,16 @@ pub(crate) fn handle_connection(mut commands: Commands, metadata: Res<GlobalMeta
 // System that reads from peripherals and adds inputs to the buffer
 pub(crate) fn buffer_input(
     tick_manager: Res<TickManager>,
+    metadata: Res<GlobalMetadata>,
     mut connection_manager: ResMut<ClientConnectionManager>,
+    server_input_event: Option<ResMut<Events<server::InputEvent<Inputs>>>>,
```

Comment:
Ideally it's transparent to clients whether they are publishing events in-app or if they do it over the network

Reply:
Yep, this is just for the proof-of-comcept; I'll try to hide it in the internal APIs